src/components/vuestic-components/va-avatar/VaAvatar.vue

```diff
@@ -1,26 +1,74 @@
 <template>
-  <div class="va-avatar">
-    <slot/>
+  <div class="va-avatar" :style="computedStyle">
+    <va-icon v-if="icon" :name="icon" />
+    <img v-if="src" :src="src" />
```

Comment:
How would it look if I specify both `icon` and `src` by mistake?

Reply:
fixed